Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/app/util/NamespaceUtils.java

```diff
@@ -29,7 +29,7 @@
  * <a name="examples"></a>
  * Example string format:
  * <ul>
- *     <li>global{@link Namespace#NAMESPACE_DELIMITER &lt;delimiter&gt;}child1{@link Namespace#NAMESPACE_DELIMITER &lt;delimiter&gt;}child2
+ *     <li>global{@link Namespace#NAMESPACE_DELIMITER ::}child1{@link Namespace#NAMESPACE_DELIMITER ::}child2
```

Comment:
I initially attempted to use `{@value Namespace#NAMESPACE_DELIMITER}` but unfortunately that produced `global"::"child1"::"child2` where the `"::"` were also link to the constant static field.